docs/service-brokers/docs/040-event-activation-custom-resource.md

```diff
@@ -0,0 +1,43 @@
+---
+title: EventActivation
+type: Custom Resource
+---
+
+The `eventactivations.remoteenvironment.kyma.cx` Custom Resource Definition (CRD) is a detailed description of the kind of data and the format used to create an Event Bus Subscription and to get an Event schema. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```
+kubectl get crd eventactivations.remoteenvironment.kyma.cx -o yaml
+```
+
+## Sample custom resource
+
+This is a sample resource that allows you to consume Events sent from the service with the `ac031e8c-9aa4-4cb7-8999-0d358726ffaa` ID in a `production` Namespace. 
+
+```
+apiVersion: remoteenvironment.kyma.cx/v1alpha1
+kind: EventActivation
+metadata:
+  name: "ac031e8c-9aa4-4cb7-8999-0d358726ffaa"
+  namespace: production
+spec:
+  displayName: "Orders"
+  source:
+    environment: "prod"
+    type: "commerce"
+    namespace: "com.hakuna.matata"
+```
+
+## Custom resource parameters
+
+This table lists all the possible parameters of a given resource together with their descriptions:
+
+
+| Parameter   |      Mandatory?      |  Description |
+|:----------:|:-------------:|:------|
+| **metadata.name** |    **YES**   | Specifies the name of the CR and the Remote Environment service ID. This field is also used to fetch Event schemas from the Minio storage.  |
```

Comment:
* `the ID of the Remote Environment service.`?
* `This field also fetches Event schemas from the Minio storage.`?

Reply:
1. That's a preferential change but ok 👍 
2. The field itself does not fetch anything, its value is used to fetch schemas. That's why I'd stay with the original version.